core/assets/bundles/bundle_id_ID.properties

```diff
@@ -146,21 +146,21 @@ mod.multiplayer.compatible = [gray]Bisa Digunakan dalam Multiplayer
 mod.disable = Nonaktifkan
 mod.content = Konten:
 mod.delete.error = Tidak dapat menghapus mod. File mungkin sedang digunakan.
-mod.incompatiblegame = [red]Outdated Game
-mod.incompatiblemod = [red]Incompatible
-mod.blacklisted = [red]Unsupported
+mod.incompatiblegame = [red]Game kadaluarsa
+mod.incompatiblemod = [red]Tidak kompatibel
+mod.blacklisted = [red]Tidak didukung
 mod.unmetdependencies = [red]Unmet Dependencies
 mod.erroredcontent = [scarlet]Konten Mengalami Kesalahan
 mod.circulardependencies = [red]Circular Dependencies
-mod.incompletedependencies = [red]Incomplete Dependencies
+mod.incompletedependencies = [red]Ketergantungan tidak lengkap
```

Comment:
dependen is more accurate; ketergantungan refers to individual's mental state

Reply:
Fix, I also changed the one below it